examples/trusted_information_retrieval/module/rust/src/lib.rs

```diff
@@ -85,11 +132,24 @@ impl TrustedInformationRetrieval for TrustedInformationRetrievalNode {
     ) -> grpc::Result<ListPointsOfInterestResponse> {
         debug!("Received request: {:?}", request);
         let location = request.location.ok_or_else(|| {
-            let err = "Location is not specified".to_string();
-            error!("Bad request: {:?}", err);
-            grpc::build_status(grpc::Code::InvalidArgument, &err)
+            let error = "Location is not specified".to_string();
+            error!("Bad request: {:?}", error);
+            grpc::build_status(grpc::Code::InvalidArgument, &error)
```

Comment:
May be clearer to build the status first, and then log it as it is (so that even the code is logged)